### PR TITLE
Ft interactive list

### PR DESCRIPTION
**Pull request by aristides1000**
I finished the following points:

- Add a new JavaScript files and import it as a module:
  - it will contain methods related to the status updates (completed: true / false).
- Add event listener to the checkbox (change).
- Update items object's value for completed key upon user actions.
- Store the updated array of items in local storage, so the user gets the correct list values after the page reloads.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,21 +1,54 @@
...
+/* eslint-disable import/no-cycle */
 /* eslint-disable no-plusplus */
 /* eslint-disable no-unused-vars */
 import _ from 'lodash';
 import './style.css';
+import { checked, textDecorationChecked, changeState } from './checked.js';
 
 const ul = document.querySelector('.task-list');
 
-const activities = [{ description: 'example 1', completed: false, index: 0 }, { description: 'example 2', completed: false, index: 1 }, { description: 'example 3', completed: false, index: 2 }];
```
Comment (dasileker) [inline src/index.js:8]: - [x] The array of object shouldn't be removed, it should be there and add the function to the array see the number 4 in the requirements 👇 it stays the array should be stored, and when I check the check box It should update from false to true see number 1 and 2
,,,,,,,llll

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @aristides1000,** 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by aristides1000** [comment]
Fix LocalStorage issues

**Review by descholar-ceo — APPROVED** [review]
Hi @aristides1000 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**